docs/README.md

```diff
@@ -437,6 +471,29 @@ Error message: ..
 To fix the above failure, you need to modify the chart as per the failure
 messages.
 
+## Signed chart failures
+
+Signed chart failures can occur when a base 64 encoded pgp public key is included in the OWNERS file.
```

Comment:
I think using `base64` without a space would be better

Reply:
Fixed.